blocks/components/editable/index.js

```diff
@@ -154,6 +172,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
+
+			if ( state !== currentState ) {
+				if ( state ) {
+					this.editor.focus();
```

Comment:
Minor: If the `focus` applies first to both `if` and `else`, might as well move it above the condition (single occurrence).